src/plugins/intel_cpu/src/mkldnn_node.cpp

```diff
@@ -573,8 +573,15 @@ void MKLDNNNode::initSupportedPrimitiveDescriptors() {
     if (!supportedPrimitiveDescriptors.empty())
         return;
 
+    auto attr = initPrimitiveAttr();
```

Comment:
Point for discussion
In the previous version attributes have not been provided to createPrimitiveDescriptorIterator
Not sure if it was intended, but as far as I understand it resulted in extra primitive descriptors being created, because extra attributes basically limit the amount of supported primitive descriptors for particular configuration.

Reply:
reverted just in case